src/__init__.py

```diff
@@ -1,34 +1,52 @@
 import os
+import json
 
 from flask import Flask
 from flask_migrate import Migrate
 from dotenv import load_dotenv
 
 from src.models import db
-from src.user import usr
-from src.config.config import TestingConfig
+from src.google import google_bp
+from src.manage import create_tables
 
 load_dotenv()
 
+def get_db_url():
+    # Heroku hack
+    db_url = str(os.environ.get('DATABASE_URL'))
+    if db_url.startswith('postgres://'):
+        db_url = db_url.replace('postgres://', 'postgresql://',1)
+    return db_url
+
+def get_secret():
+    SECRET_KEY = os.environ.get('SECRET_KEY')
+    if not SECRET_KEY:
+        return 'S9M3_SeCR3T'
```

Comment:
No if there is no secret key, return an error. This should actually be done in `create_app` function instead. You don't need a function for this.